### ISSUE TITLE
Overwriting plot visualization

### ISSUE THREAD

**Issue by acosta001** [nathandunk/BetterSerialPlotter#11]
Hi, I am looking for a serial plotter with an "overwriting" style, meaning that when the x axis reaches the end it restarts from left and overwrites the previous plot. In some cases, this would help the readability of the data. Here is an [example](https://youtu.be/zN2a8k-Qdb8).
Is there a way to do it with BetterSerialPlotter? I would love it. Thanks.

**Comment by nathandunk** [comment]
Yes, that shouldn't be too hard to add. Let me work on it and see if I can send you an attempt and see if it meets what you are looking for!

**Comment by acosta001** [comment]
Great! Thank you very  much

**Comment by acosta001** [comment]
Hi! By any chance, did you manage to implement the "overwriting style" of plot visualization? If yes, I would be glad to try it. Thank you very much!

**Comment by LimBingSheng** [comment]
I also need that too! As I am plotting a real time fft graph. I will be greatly thankful if you can done it. Thanks!
